test/e2e/framework/ci-operator.go

```diff
@@ -131,8 +139,11 @@ func (c *CiOperatorCommand) Run() ([]byte, error) {
 		c.cleanupDone <- struct{}{}
 	}
 	// TODO(skuznets): stream this output?
-	err := c.cmd.Wait()
-	output := b.Bytes()
+	err = c.cmd.Wait()
+	output, readErr := ioutil.ReadAll(tee)
```

Comment:
Why-read it, can't we simply get it from the `log` buffer?

Reply:
This is a convenient way to write to the file as we read from it

Reply:
It's not a re-read and is efficient

Reply:
ah sorry, I read the ioutil and though you were reading from the file.

You need to do a `defer logFile.Close()` though